src/sound/jam_engine.rs

```diff
@@ -217,6 +226,19 @@ impl JamEngine {
             // println!("mixer: {}", self.mixer);
         }
     }
+    // This is where we check for a new pedalboard
+    fn check_pedal_board(&mut self) -> () {
+        match self.pedal_rx.try_recv() {
+            Ok(board) => {
+                let idx = board.get_channel();
+                if idx < 2 {
+                    self.pedal_boards[idx] = board;
+                }
+                self.send_pedal_info();
```

Comment:
send_pedal_info is still using the status_dx channel. Should it use the PedalBoard channel instead now? I'm not sure given that I don't know the full context, so just asking if that makes more sense. Related to that, is there any other info that needs to be sent to the PedalBoard channel in send_pedal_info to reflect the new design?

Reply:
I think after the PedalBoard is sent to the jack thread via the mpc::channel the building thread no longer has access to it (Rust borrow rules) so it cant really send the info.  Plus the PedalInfo is for both channels' boards so it needs to stay in the jack thread.  I don't think building the json is that intense but it is something to watch.